### ISSUE TITLE
Custom commands are executed in the local terminal instead of the ssh-connection

### ISSUE THREAD

**Issue by jan-di** [VitalyKondratiev/vscode-sshextension#19]
The configured custom commands are not entered into the ssh session. They are entered in the terminal, after the ssh session is closed by the user and so are executed locally instead of on the remote server.

Config:
```json
{
    "name": "<host>",
    "host": "<host>",
    "port": 22,
    "username": "<user>",
    "customCommands": [
        "ls"
    ]
}
```

Output:
```powershell
~\$ ssh <host> -l <user>
Last login: Sat Mar  2 23:53:22 2019 from ****
<user>@<host>:~ $
<user>@<host>:~ $
<user>@<host>:~ $ exit
logout
Connection to <host> closed.
~\$ ls
# local output of ls
```

VSCode 1.31.1
Integrated Terminal: PowerShell Core 6.1.2
Microsoft Windows [Version 10.0.17134.590] (#1803)
OpenSSH_for_Windows_7.6p1, LibreSSL 2.6.4

**Comment by BernierCR** [comment]
Yes, this would be great if someone would fix it. Without the custom commands working, this extension has minimal value, still some value, but more with them working.

**Comment by mat01** [comment]
Same for me - custom commands are seen executed locally, and the 'path' directive does nothing.
